### ISSUE TITLE
Mysterious documentation

### ISSUE THREAD

**Issue by Qoraiche** [spatie/laravel-medialibrary-pro-app#21]
The documentation seems to be a little bit mysterious and hard to follow, especially for people with less experience, anyway in the doc > Handling uploads with Blade > is mandatory to install liveware? what is only want to use a blade component and embed the compiled js file? and why you don't put the full instruction to make it work with livewire instead of putting a link to install it.

**Comment by freekmurze** [comment]
It is mandatory to install Livewire, as the upload functionality of the Blade components is provided through Livewire.

We link to the Livewire installation instruction so that when they update those instructions, we don't have to update our instructions.

If you're not familiar with Livewire yet, I can recommend trying it out so you get a feel of what it is capable of. Our Blade components are basically wrappers around the upload functionality provided by Livewire (https://laravel-livewire.com/docs/2.x/file-uploads)
